app/views/pages/dashboard.html.erb

```diff
@@ -11,10 +11,14 @@
     </ul>
   </nav>
 
+   <!-- @services is no longer an array after I pulled from main branch. cannot be itterated over -->
```

Comment:
Not sure how to reconcile this comment with what's happening on line 19. Seems like you're still iterating over the services array.